app/views/review_mapping/_review_report.html.erb

```diff
@@ -16,12 +16,18 @@
         <th width = "24%" class="sorter-false">Team reviewed <span></span></th>
       <% end %>
       <th width = "13%" class="sorter-false">Score awarded / Avg. score <span></span></th>
-      <th width = "13%" class="sorter-true">Metrics <span></span></th>
+      <% if @metric_type == "Sentiment" %>
```

Comment:
Shouldn't the name of the metric be retrieved from the db, rather than included as a text string in the code?  Every time a new metric is added, this code will have to be changed too.